### PR TITLE
Updated getting started links on docs home page

### PR DESCRIPTION
**Pull request by olehermanse**
The getting started links currently point to some of the old installation instructions, etc.

Better to highlight the new content.

### CODE REVIEW

**Comment by olehermanse** [comment]
@cf-bottom jenkins, please :)

**Review by aleksandrychev — APPROVED** [review]

**Review by chiachenglu — APPROVED** [review]

**Comment by nickanderson** [comment]
In the preview build I get  404.

```
Not Found

The requested URL /packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-196/output/_site/getting-started-writing-and-serving-policy.html was not found on this server.
```

**Review by nickanderson — DISMISSED** [review]
The link for Reporting and Web UI gives me a 404.

File: index.markdown [hunk 1]
```diff
@@ -21,28 +21,28 @@ alias: index.html
...
          <li>
-            <a href="getting-started-installation-general-installation.html">General Installation</a>
-            <div>The Steps to bring up a CFEngine installation within an organization.</div>
+            <a href="getting-started-modules-from-cfengine-build.html">Modules</a>
+            <div>Use modules to easily add reports or get things done without writing any code.</div>
          </li>
          <li>
-            <a href="getting-started-writing-and-serving-policy.html">Writing and serving policy</a>
-            <div>Create promises which defines how some part of an overall system should behave.</div>
+            <a href="getting-started-writing-and-serving-policy.html">Reporting and Web UI</a>
```
Comment (olehermanse) [inline index.markdown:32]:
```suggestion
            <a href="getting-started-reporting-and-web-ui.html">Reporting and Web UI</a>
```

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/197/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/197/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-197/output/_site/

**Pull request merged**